products/compute/terraform.yaml

```diff
@@ -543,7 +543,14 @@ overrides: !ruby/object:Overrides::ResourceOverrides
         required: false
         default_from_api: true
     custom_code: !ruby/object:Provider::Terraform::CustomCode
+      extra_schema_entry: templates/terraform/extra_schema_entry/compute_network_delete_default_route.erb
+      post_create: templates/terraform/post_create/compute_network_delete_default_route.erb
+      post_import: templates/terraform/custom_import/compute_network_delete_default_route.erb
       encoder: templates/terraform/encoders/network.erb
+    docs: !ruby/object:Provider::Terraform::Docs
+      optional_properties: |
+        * `delete_default_route`: If set to `true`, the default route (`0.0.0.0/0`) will be deleted
```

Comment:
What do you think of naming this `delete_default_routes_on_create`? That makes it super clear that an update won't delete routes, and spells out the fact that we delete all routes and not just a single default one in case it becomes true that there are more than one.

Reply:
Done.